src/sass/card/card.scss

```diff
@@ -0,0 +1,137 @@
+@import "../mixins/private/bubble-mixins";
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.card {
+    display: flex;
+    flex-direction: column;
+    width: $card-size-default;
+
+    &--large {
+        width: $card-size-large;
+    }
+
+    &__container {
+        position: relative;
+        height: $card-size-default;
+
+        &--large {
+            height: $card-size-large;
+        }
+    }
+
+    &__content {
+        display: flex;
+        height: inherit;
+        width: inherit;
+        justify-content: center;
+        align-items: center;
+        box-sizing: border-box;
+        background-color: var(--color-background-secondary, #f7f7f7);
+        border-radius: var(--border-radius-100, 16px);
+        padding: var(--spacing-150, 12px);
+
+        .progress-bar {
+            width: 100%;
+        }
+
+        &--image {
+            border-radius: var(--border-radius-100, 16px);
```

Comment:
Is there ever a `card__content--image` that isn't also `card__content`? Looks like this `border-radius` is redundant.

Reply:
Thanks, will fix

Reply:
I also refactored the `file-upload` component as well. Now if you add `--compact` respective class to the root component it will handle the compact styles in nested children as needed

Reply:
I removed the need for `card__content--image`, if the component adds the background-image then it will size it properly without the need for this class